missing script: package

When I [attempt to run `npm run package`](https://github.com/adam7/platformsh-cli-action/pull/11#issuecomment-804784297) I get the following error:

```
npm ERR! missing script: package
```

--- a/package.json
+++ b/package.json
@@ -6,7 +6,8 @@
   "main": "lib/main.js",
   "scripts": {
     "build": "tsc",
-    "test": "jest"
+    "test": "jest",
+    "package": "ncc build --source-map --license licenses.txt"
   },
   "repository": {
     "type": "git",

